Libraries/Boards/MAX32690/APARD/Include/board.h

```diff
@@ -0,0 +1,125 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2022-2023 Maxim Integrated Products, Inc. (now owned by
+ * Analog Devices, Inc.),
+ * Copyright (C) 2023-2024 Analog Devices, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+/**
+ * @file    board.h
+ * @brief   Board support package API.
+ */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifndef LIBRARIES_BOARDS_MAX32690_APARD_INCLUDE_BOARD_H_
+#define LIBRARIES_BOARDS_MAX32690_APARD_INCLUDE_BOARD_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define BOARD_APARD
+
+#ifndef CONSOLE_UART
+#define CONSOLE_UART 0 /// UART instance to use for console
+#endif
+
+#ifndef CONSOLE_BAUD
+#define CONSOLE_BAUD 115200 /// Console baud rate
+#endif
+
+#ifndef HCI_UART
+#define HCI_UART 3
+#endif
+
+#ifndef TERMINAL_UART
+#define TERMINAL_UART CONSOLE_UART
+#endif
+
+#ifndef USER_UART
+#define USER_UART 1
+#endif
+
+#ifdef LED_OFF
+#undef LED_OFF
+#endif
+#define LED_OFF 0 /// Override inactive state of LEDs
+
+#ifdef LED_ON
+#undef LED_ON
+#endif
+#define LED_ON 1 /// Override active state of LEDs
+
+#ifndef EXT_FLASH_BAUD
+#define EXT_FLASH_BAUD 4000000
+#endif
+
+#define APARD /// Used in examples to control program flow
```

Comment:
Just noticed this (minor quibble) - `BOARD_APARD` is defined on line 37.  Existing conventions prefers using `BOARD_xxx` for program flow

Reply:
Updated as requested.